human-tracking-and-pose-detection/.gitignore

```diff
@@ -0,0 +1,3 @@
+
```

Comment:
This package should go inside `perception`. I'd probably name it something more concise than `human-tracking-and-pose-detection` as well